src/Symfony/Component/HttpKernel/CHANGELOG.md

```diff
@@ -1,6 +1,11 @@
 CHANGELOG
 =========
 
+8.1
+---
+
+ * Deprecate the `Extension` class, use the `Extension` class from the DependencyInjection component instead
```

Comment:
Using FQCN is more explicit here because the classes have the same short name.
```suggestion
 * Deprecate the `Symfony\Component\HttpKernel\DependencyInjection\Extension` class, use the parent `Symfony\Component\DependencyInjection\Extension\Extension` instead
```